test/functional/p2p_invalid_messages.py

```diff
@@ -66,7 +67,10 @@ def run_test(self):
         msg_at_size = msg_unrecognized("b" * valid_data_limit)
         assert len(msg_at_size.serialize()) == msg_limit
 
-        with node.assert_memory_usage_stable(perc_increase_allowed=0.03):
+        perc_increase_allowed = 0.03
+        if [s for s in os.environ.get("BITCOIN_CONFIG", "").split(" ") if "--with-sanitizers" in s and "address" in s]:
+            perc_increase_allowed = 3.5
+        with node.assert_memory_usage_stable(perc_increase_allowed=perc_increase_allowed):
```

Comment:
I think the test fails unrelated to the sanitizer?

Reply:
I'm not so sure about that: could it be that this failure is intermittent without ASAN and consistent with ASAN? :-) It appears to be consistently triggered under ASAN for me.